sentry-android-core/src/test/java/io/sentry/android/core/internal/util/ContentProviderSecurityCheckerTest.kt

```diff
@@ -0,0 +1,107 @@
+package io.sentry.android.core.internal.util
+
+import android.content.ContentProvider
+import android.content.Context
+import android.os.Build
+import io.sentry.android.core.BuildInfoProvider
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verifyNoInteractions
+import org.mockito.kotlin.whenever
+import kotlin.test.Test
```

Comment:
```suggestion
import kotlin.test.Test
import kotlin.test.assertFailsWith
```

Reply:
@markushi, ahh! This is what I've been looking for! Thanks!

I will use `assertFailsWith` 🔥

Reply:
Done! https://github.com/getsentry/sentry-java/pull/2466/commits/b7a40d7c982af216508034960545e7402ec6b3f5